### PR TITLE
Altered sim_vars to output data.frame with units and longname

### PR DESCRIPTION
**Pull request by lawinslow**

### CODE REVIEW

**Comment by lawinslow** [comment]
@jread-usgs I'll let you merge.

**Comment by jread-usgs** [comment]
say what #judgementbot?

File: R/sim_var_longname.R [hunk 1]
```diff
@@ -9,7 +9,7 @@
 #'sim_folder <- run_example_sim(verbose = FALSE)
 #'nc_file <- file.path(sim_folder, 'output.nc')
 #'vars <- sim_vars(file = nc_file)
-#'sim_var_units(nc_file, vars[1])
+#'sim_var_units(nc_file, vars[1,]$name)
```
Comment (jread-usgs) [inline R/sim_var_longname.R]: whoops. I guess this example doesn't actually use the sim_var_longname function

Comment (lawinslow) [inline R/sim_var_longname.R]: ha, I'll fix

**Pull request merged**